src/lib389/lib389/monitor.py

```diff
@@ -164,68 +164,64 @@ class MonitorLDBM(DSLdapObject):
         :type instance: lib389.DirSrv
         :param dn: not used
     """
+    DB_KEYS = {
+        DB_IMPL_BDB: [
+            'dbcachehits', 'dbcachetries', 'dbcachehitratio',
+            'dbcachepagein', 'dbcachepageout', 'dbcacheroevict',
+            'dbcacherwevict'
+        ],
+        DB_IMPL_MDB: [
+            'normalizeddncachetries', 'normalizeddncachehits',
```

Comment:
A bit surprised: Should not the dn cache statistic be visible in bdb too ?

Reply:
The normalizeddncache... keys for a bdb implementation are extended in MonitorLDBM.__init, but only if the version is newer than 1.4.0